SpywareFilter/sections/specific.txt

```diff
@@ -144,8 +144,11 @@ fategrandorder.info##.sidebar_acr.side_inline
 ||dojq4kt8ws9iq.cloudfront.net/js/analytics.js
 ||components2.gazzettaobjects.it/rcs_tealium/
 ||liquipedia.net/stats/
-||static.olx.com.br/olx/js/lurker.min.js
+||lurker.olx.com.br^
+||geoip.olx.com.br^
```

Comment:
not tracking. Look request content.

Reply:
geoip.olx.com.br/ip identifies the user's most accurate location and displays ads specific to that region. The second screenshot shows this (obviously I hid my location)

Reply:
It uses info about IP. They know it in any case. This is used not for tracking.

Reply:
Yes, of course. but blocking the ip will prevent the site from displaying targeted advertising. Especially when using the mobile app, as it's not possible to apply css filters inside it.

Reply:
You [probably] will see just non-relevant ads(created by users on this site).